tests/codegen-llvm/align-byval.rs

```diff
@@ -1,7 +1,7 @@
 // ignore-tidy-linelength
 //@ add-minicore
 //@ revisions:m68k x86_64-linux x86_64-windows i686-linux i686-windows
-
+//@ compile-flags: -Copt-level=1 -Cno-prepopulate-passes
```

Comment:
This test now relies on CopyProp + DeadStoreElimination, while it previously relied only on CopyProp.

Custom MIR would be a better alternative, if not for the fact that it is not available in minicore tests :-(.

Reply:
Ah I see. I'm just wary of the assumption that opt-level 1 is _fully_ optimized MIR. Currently it does map to -Zmir-opt-level=2 but I have a nagging feeling that enabling the same (with the exception of Inline) MIR opts at -Copt-level=3 and -Copt-level=1 can't be optimal.

But this isn't a serious reason to hold up this PR even more, it's an important fix so I'm delta-approving it.